src/components/AuthWrappers/EnforceLoginStatePageWrapper.tsx

```diff
@@ -15,17 +15,17 @@ interface EnforceLoginStatePageWrapperProps {
   redirectTo?: string
 }
 
-const Redirect = ({ redirectTo }: EnforceLoginStatePageWrapperProps) => {
+const Redirect = ({ redirectTo = SIGN_IN }: EnforceLoginStatePageWrapperProps) => {
   const router = useRouter()
   const redirectUrl = useMemo(() => {
     if (typeof window === 'undefined') return encodeURIComponent('/')
```

Comment:
Does this line need to change too? Also uses `encodeURIComponent`

Reply:
Yes, sorry I missed that.